### PR TITLE
[Refactor] Enable help and code refactor on several scripts

### PR DESCRIPTION
**Pull request by Deegue**
closes https://github.com/intel/llm-on-ray/issues/35

- Include --help (add_help=True)
- Command line description for query_single.py: "Example script to query with single request"
- Remove query_batch.py since it just used RayData to store prompts, it's not fit for our serving scenarios any longer.
- Remove time measurement as we will add specific benchmark program later

### CODE REVIEW

File: ui/start_ui.py [hunk 1]
```diff
@@ -864,7 +864,7 @@ def _init_ui(self):
         self.gr_chat = gr_chat
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description="Start UI", add_help=False)
+    parser = argparse.ArgumentParser(description="Start UI", add_help=True)
```
Comment (xwu99) [inline ui/start_ui.py:867]: Web UI for finetuning and deploying LLMs

**Comment by xwu99** [comment]
@KepingYan query_batch.py is removed here.

File: ui/start_ui.py (hunk 1 above)
Comment (KepingYan) [reply ui/start_ui.py:867]: How about changing it to `Web UI` as introduced in docs?

File: inference/serve.py [hunk 2]
```diff
@@ -58,22 +58,22 @@ def get_deployed_models(args):
...
+    parser.add_argument("--tokenizer", default=None, type=str, help="Tokenizer name or path.")
+    parser.add_argument("--port", default=8000, type=int, help="The port of deployment address.")
+    parser.add_argument("--route_prefix", default=None, type=str, help="The route prefix for HTTP requests.")
+    parser.add_argument("--cpus_per_worker", default="24", type=int, help="CPUs per worker.")
+    parser.add_argument("--gpus_per_worker", default=0, type=float, help="GPUs per worker, used when --device is cuda.")
+    parser.add_argument("--hpus_per_worker", default=0, type=float, help="HPUs per worker, used when --device is hpu.")
+    parser.add_argument("--deepspeed", action='store_true', help="Enable deepspeed inference.")
+    parser.add_argument("--workers_per_group", default="2", type=int, help="Workers per group, used with --deepspeed.")
+    parser.add_argument("--ipex", action='store_true', help="Enable ipex optimization.")
+    parser.add_argument("--device", default="cpu", type=str, help="CPU, XPU, HPU or CUDA.")
```
Comment (KepingYan) [inline inference/serve.py:73]: Please keep uppercase and lowercase consistent here

Comment (Deegue) [reply inference/serve.py:73]: I think these are shortcuts and maybe should be uppercase.

File: ui/start_ui.py (hunk 1 above)
Comment (Deegue) [reply ui/start_ui.py:867]: Revised.

File: inference/serve.py (hunk 2 above)
Comment (xwu99) [reply inference/serve.py:73]: > I think these are shortcuts and maybe should be uppercase.

You need to check back to the code if this is used as device string and if the device string is case sensitive.

Comment (Deegue) [reply inference/serve.py:73]: Got it. I found some places use uppercase like [L60](https://github.com/intel/llm-on-ray/blob/87ffd4f92336e49fcb54882ff331b9cb6aa6ef79/finetune/finetune_config.py#L60C33-L60C33) and some use lowercase [L217](https://github.com/intel/llm-on-ray/blob/87ffd4f92336e49fcb54882ff331b9cb6aa6ef79/finetune/finetune.py#L217C8-L217C8)...

Comment (Deegue) [reply inference/serve.py:73]: Also changed default to uppercase, to see if somewhere will go wrong.

Comment (xwu99) [reply inference/serve.py]: > Also changed default to uppercase, to see if somewhere will go wrong.

Pls change default to lowercase as presented as torch device. 
https://pytorch.org/docs/stable/tensor_attributes.html#torch-device

Comment (xwu99) [reply inference/serve.py]: > Got it. I found some places use uppercase like [L60](https://github.com/intel/llm-on-ray/blob/87ffd4f92336e49fcb54882ff331b9cb6aa6ef79/finetune/finetune_config.py#L60C33-L60C33) and some use lowercase [L217](https://github.com/intel/llm-on-ray/blob/87ffd4f92336e49fcb54882ff331b9cb6aa6ef79/finetune/finetune.py#L217C8-L217C8)...

changed to lowercase as torch device, I would suggest use lowercase directly
https://github.com/intel/llm-on-ray/blob/87ffd4f92336e49fcb54882ff331b9cb6aa6ef79/finetune/finetune.py#L216

Comment (Deegue) [reply inference/serve.py]: Ok, changed to lowercase.

Comment (xwu99) [reply inference/serve.py]: please click resolve conversation above and below.

**Review by xwu99 — APPROVED** [review]
@Deegue approved. pls merge after CI passed

**Comment by Deegue** [comment]
Thanks @xwu99 , merged.

**Pull request merged**